pages/1.14/multi-tenancy/resource-mgmt-primitives/index.md

```diff
@@ -0,0 +1,395 @@
+---
+layout: layout.pug
+navigationTitle: Resource Management Primitives
+title: Resource Management Primitives
+menuWeight: 10
+excerpt: Reserving resources to support multi-tenancy
+render: mustache
+model: /1.14/data.yml
+---
+Resources in DC/OS can be reserved and prioritized using a combination of roles, reservations, quotas, and weights. These features are provided by Apache Mesos, at the core of DC/OS and are referred to as `Primitives`, as they are only accessible via the API and have not yet been integrated into the DC/OS UI or CLI. A user requires good monitoring in place of available/used resources when working with quotas, reservations, and weights.
```

Comment:
minor comment. 
"as they are only accessible" --> "as most of them are only accessible"